packages/ilib-loctool-android-layout/package.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "ilib-loctool-android-layout",
+    "version": "1.0.3",
+    "main": "./AndroidLayoutFileType.js",
+    "description": "A loctool plugin that knows how to process Android Layout Files",
+    "license": "Apache-2.0",
+    "keywords": [
+        "internationalization",
+        "i18n",
+        "localization",
+        "l10n",
+        "globalization",
+        "g11n",
+        "strings",
+        "resources",
+        "locale",
+        "translation",
+        "webpack",
+        "loader",
+        "android",
+        "layout"
+    ],
+    "email": "ehoogerbeets@gmail.com",
+    "author": {
+        "name": "Edwin Hoogerbeets",
+        "web": "http://www.translationcircle.com/",
+        "email": "ehoogerbeets@gmail.com"
+    },
+    "contributors": [
+        {
+            "name": "Edwin Hoogerbeets",
+            "email": "ehoogerbeets@gmail.com"
+        }
+    ],
+    "files": [
+        "README.md",
+        "LICENSE",
+        "AndroidLayoutFileType.js",
+        "AndroidLayoutFile.js"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/iLib-js/ilib-mono/packages/ilib-loctool-android-layout.git"
```

Comment:
Not sure but I think this is not a valid git URL. It should probably be just `https://github.com/iLib-js/ilib-mono.git`

Reply:
note: this applies to all packages in this batch